### PR TITLE
Ramico/private props

### PR DESCRIPTION
**Pull request by ramico**

### CODE REVIEW

File: includes/class-wc-siftscience-comm.php [hunk 1]
```diff
@@ -19,21 +19,21 @@ class WC_SiftScience_Comm {
 		private const DELETE_URL = 'https://api.sift.com/v204/users/{user}/labels/?api_key={api}&abuse_type=payment_abuse';
 		private const SCORE_URL = 'https://api.sift.com/v204/score/{user}/?api_key={api}';
 
-		private $options;
-		private $logger;
+		private $_options;
+		private $_logger;
 
 		private $headers = array(
```
Comment (nabsul) [inline includes/class-wc-siftscience-comm.php:25]: I guess this should be changed with underscore or even const if it's possible for arrays.

File: includes/class-wc-siftscience-events.php [hunk 2]
```diff
@@ -208,7 +208,7 @@ public function send_queued_data() {
 
 			foreach ( $this->events as $event ) {
 				if ( null != $event ) {
-					$this->comm->post_event( $event );
+					$this->_comm->post_event( $event );
 				}
 			}
```
Comment (nabsul) [inline includes/class-wc-siftscience-events.php:214]: I think `events` and `order_map` should be declared and have underscores.

File: includes/class-wc-siftscience-format.php [hunk 3]
```diff
@@ -22,20 +22,20 @@
 	require_once( 'class-wc-siftscience-format-transaction.php' );
 
 	class WC_SiftScience_Format {
-		public $items;
-		public $login;
-		public $account;
-		public $order;
-		public $cart;
-		public $transactions;
+		public $_items;
```
Comment (nabsul) [inline includes/class-wc-siftscience-format.php:25]: public props should not have underscore in the name

File: includes/class-wc-siftscience-comm.php (hunk 1 above)
Comment (ramico) [reply includes/class-wc-siftscience-comm.php:25]: didn't c that

File: includes/class-wc-siftscience-events.php (hunk 2 above)
Comment (ramico) [reply includes/class-wc-siftscience-events.php:214]: there were declared way below wait next commit

File: includes/class-wc-siftscience-format.php (hunk 3 above)
Comment (ramico) [reply includes/class-wc-siftscience-format.php:25]: i read it private sorry

**Comment by ramico** [comment]
worth noting 
As of PHP 5.6 it is possible to define array constant

**Pull request merged**